udata/templates/search.html

```diff
@@ -78,6 +80,7 @@
 {% endblock %}
 
 {% block toolbar_container %}
+{% if datasets %}
```

Comment:
Toolbars are always rendered, just not visible.
The current visible is linked in JS in https://github.com/etalab/udata/blob/master/js/search.js#L16-L27